active-tasks/react_routing.json

```diff
@@ -81,6 +86,11 @@
       "type": "penalty",
       "text": "Commits after the deadline",
       "max": -40
+    },
+    {
+      "type": "penalty",
+      "text": "Pull Request doesn't follow guideline (including checkboxes in Score) [PR example](https://docs.rs.school/#/en/pull-request-review-process?id=pull-request-description-must-contain-the-following)",
```

Comment:
Not sure if it supports `md` syntax, so let's check it out

Reply:
No, it doesn't https://discord.com/channels/794806036506607647/801113609752412220/1262297502376263703